src/System.Json/tests/JsonValueTests.cs

```diff
@@ -88,6 +88,8 @@ public void JsonValue_Load_ToString_JsonArrayWithNulls()
                 Assert.Equal(4, value.Count);
                 Assert.Equal(JsonType.Array, value.JsonType);
                 Assert.Equal("[1, 2, 3, null]", value.ToString());
+                ((JsonArray) value).Add(null);
+                Assert.Equal("[1, 2, 3, null, null]", value.ToString());
```

Comment:
The Add_NullItem_ThrowsArgumentNullException test will also need to be fixed, as it's explicitly checking for this exception.